packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -3,26 +3,39 @@ import { addDisposer, getRoot, isValidReference, onAction, types } from 'mobx-st
 
 import Step from './Step'
 import { DrawingTask, DataVisAnnotationTask, MultipleChoiceTask, SingleChoiceTask } from './tasks'
+import { TextModel } from '@plugins/tasks/TextTask'
+
+function taskDispatcher (snapshot) {
+  if (snapshot.type === 'drawing') return DrawingTask
+  if (snapshot.type === 'multiple') return MultipleChoiceTask
+  if (snapshot.type === 'single') return SingleChoiceTask
+  if (snapshot.type === 'dataVisAnnotation') return DataVisAnnotationTask
+  if (snapshot.type === 'text') return TextModel
```

Comment:
Just a minor quibble on naming here, but maybe `TextTaskModel` just for clarity?

Reply:
Yeah, the naming is not consistent at all, particularly where models and views have the same name.

Reply:
The worst example might be `DataVisAnnotation` and `DataVisAnnotationTask`.

Reply:
Naming is hard